com.ibm.wala.cast.python.test/data/tf2_test_callbacks2.py

```diff
@@ -0,0 +1,11 @@
+import tensorflow as tf
+
+# Code from https://www.tensorflow.org/api_docs/python/tf/distribute/Strategy#example_usage_2
```

Comment:
Why are we not using a version in this URL?

Reply:
It is the same example as in the link.